pkg/trace/stats/aggregation.go

```diff
@@ -64,8 +80,31 @@ func getStatusCode(s *pb.Span) uint32 {
 }
 
 // NewAggregationFromSpan creates a new aggregation from the provided span and env
-func NewAggregationFromSpan(s *pb.Span, origin string, aggKey PayloadAggregationKey, enablePeerSvcAgg bool) Aggregation {
+func NewAggregationFromSpan(s *pb.Span, origin string, aggKey PayloadAggregationKey, enablePeerSvcAgg bool, customTagConf map[string][]string) Aggregation {
 	synthetics := strings.HasPrefix(origin, tagSynthetics)
+
+	customTagString := []string{}
+
+	spanTags, ok := customTagConf[s.Name]
+
+	if ok {
+		for k := range spanTags {
+			customTagString = append(customTagString, spanTags[k]+":"+s.Meta[spanTags[k]])
```

Comment:
If `spanTags[k]` is not in the meta, do we really want to add something like `costcenter:`?

Why not just add the tags that exist?